### PR TITLE
More proper Styling

### PR DESCRIPTION
**Pull request by larsblumberg**
- room titles have background color
- section (right page) has top/bottom padding
- margin below "add devices" bottom
- proper font size for device columns
- removed navigation's bar hover color
- centered top navigation bar and set its width to 80%

### CODE REVIEW

File: public/app/project-setup/theme/components/_grid.scss [hunk 1]
```diff
@@ -14,3 +14,9 @@ $column-height: $column-width;
 .grid-14 {
   @include grid-generator($section-columns);
 }
+
+.row {
+  &.header {
+    background: $passive-gray;
+  }
+}
```
Comment (DaQuirm) [inline public/app/project-setup/theme/components/_grid.scss]: Why have you done that in this way?

File: public/app/project-setup/theme/mixins/_grid-generator.scss [hunk 2]
```diff
@@ -88,10 +89,10 @@
   }
 
   &.padded {
-    padding: 0 $basic-padding;
+    padding: $basic-padding;
```
Comment (DaQuirm) [inline public/app/project-setup/theme/mixins/_grid-generator.scss]: Why?

File: public/app/screens/main/navigation/navigation.jsx [hunk 3]
```diff
@@ -19,8 +19,8 @@ var Navigation = React.createClass({
 
       return (
         <Link to={item.linkTo} key={i} className={classNames}>
-          <i className={'cc_navigation__linkImage' + ' show-for-landscape ' + 'icon-' + item.image}></i>
-          <label className="cc_navigation__linkLabel">{item.title}</label>
+          <i className={'cc_navigation__link-image' + ' show-for-landscape' + ' icon-' + item.image}></i>
+          <label className="cc_navigation__link-label">{item.title}</label>
```
Comment (DaQuirm) [inline public/app/screens/main/navigation/navigation.jsx]: Why?

File: public/app/screens/main/navigation/style.scss [hunk 4]
```diff
@@ -10,25 +10,33 @@
...
+  &.horizontal {
+    width: 80%;
+  }
+
   & * {
     cursor: pointer;
   }
 
+  > a:hover {
+    background: inherit;
```
Comment (DaQuirm) [inline public/app/screens/main/navigation/style.scss]: Why did we get rid of the highlighting?

What are the reasons from designers?

File: public/app/screens/main/navigation/style.scss [hunk 5]
```diff
@@ -10,25 +10,33 @@
...
+
   &__link {
     padding: 10px 0 10px 0;
-  }
 
-  &__linkImage {
-    float: left;
-    margin: -5px 20px 0 10px !important;
-  }
+    &-image {
```
Comment (DaQuirm) [inline public/app/screens/main/navigation/style.scss]: This is a modifier. Please, refer to the coment regarding BEM styles above.

File: public/app/screens/main/navigation/style.scss [hunk 6]
```diff
@@ -10,25 +10,33 @@
...
+      float: left;
+      margin: -5px 20px 0 10px !important;
+    }
 
-  &__linkLabel {
-    font-size: $h2-font-size - $h2-font-reduction;
+    &-label {
+      font-size: $h2-font-size - $h2-font-reduction;
```
Comment (DaQuirm) [inline public/app/screens/main/navigation/style.scss]: What's with indentation?

File: public/app/screens/main/navigation/style.scss [hunk 7]
```diff
@@ -10,25 +10,33 @@
...
 
-    @media #{$medium-up} {
-      line-height: $header-line-height;
-      font-size: $h2-font-size;
+      @media #{$medium-up} {
+        line-height: $header-line-height;
+        font-size: $h2-font-size;
+      }
```
Comment (DaQuirm) [inline public/app/screens/main/navigation/style.scss]: Fix the indentation plz

File: public/app/screens/main/sections/device-list/devices.jsx [hunk 8]
```diff
@@ -29,7 +29,7 @@ var DevicesSection = React.createClass({
 
     return (
       <div className='cc-devices grid-14 padded'>
-        <DeviceFinder />
+        <DeviceFinder className='cc-devices__device-finder' />
```
Comment (DaQuirm) [inline public/app/screens/main/sections/device-list/devices.jsx]: Why don't you just add the styles to the `DeviceFinder` or create a *container* for that?

File: public/app/screens/main/sections/device-list/room/device/device.jsx [hunk 9]
```diff
@@ -14,11 +14,12 @@ var Device = React.createClass({
...
         <div className='cc-device__title columns small-4 padded-left'>
           <Link to='device' params={{deviceId: this.props.device.id}}>
-            <h5>{this.props.device.title}</h5>
+            <h4>{this.props.device.title}</h4>
           </Link>
         </div>
         <div className='cc-device__type columns small-4 text-center'>
-          <h5 className='subheader'>{this.props.device.type}</h5>
+          {/* TODO: What is 'subheader' used for? */}
+          <h4 className='subheader'>{this.props.device.type}</h4>
```
Comment (DaQuirm) [inline public/app/screens/main/sections/device-list/room/device/device.jsx]: It is used for coloring and denoting the fact, that this thing is a subheader.

**Comment by mr-mig** [comment]
The changes from this PR was integrated in the other branch. So I am closing it,

**Pull request closed without merging**